test/specs/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -85,6 +81,12 @@ describe('rules/require-padding-newlines-before-line-comments', function() {
         it('should report error if first line after a curly', function() {
             assert(checker.checkString('if (true) {\n// comment\n}').getErrorCount() === 1);
         });
+
+        it('should not consider code and comment on the same line (#1194)', function() {
+            assert(checker.checkString('var a; \n var b; //comment\n').isEmpty());
+            assert(checker.checkString('var a; \n var b; //comment\nvar c;').isEmpty());
+            assert(checker.checkString('/**/var a; \n var b// comment\n').isEmpty());
```

Comment:
ah `/* */` but without a space. Gotcha. For some reason this confused me. All is well, ignore me. :P